NickvisionCavalier.Shared/Models/CmdOptions.cs

```diff
@@ -0,0 +1,84 @@
+using CommandLine;
+
+namespace NickvisionCavalier.Shared.Models;
+
+/// <summary>
+/// Command-line options
+/// </summary>
+public class CmdOptions {
+    /// <summary>
+    /// Size of drawing area margins in pixels
+    /// </summary>
+    [Option('n', "margin", Required = false, HelpText = "Area margin (0-40)")]
+    public uint? AreaMargin { get; set; }
+    /// <summary>
+    /// Whether the window should be borderless
+    /// </summary>
+    [Option('w', "borderless", Required = false, HelpText = "Make window borderless (true or false)")]
+    public bool? Borderless { get; set; }
+    /// <summary>
+    /// Whether the corners of the window should be sharp
+    /// </summary>
+    [Option('s', "sharp-corners", Required = false, HelpText = "Make window corneres sharp (true or false)")]
+    public bool? SharpCorners { get; set; }
+    /// <summary>
+    /// Number of bar pairs in CAVA
+    /// </summary>
+    [Option('b', "bars", Required = false, HelpText = "Number of bar pairs (3-25)")]
+    public uint? BarPairs { get; set; }
+    /// <summary>
+    /// Whether to set channels to stereo (mono if false)
+    /// </summary>
+    [Option('c', "stereo", Required = false, HelpText = "Mono (false) or stereo (true)")]
+    public bool? Stereo { get; set; }
+    /// <summary>
+    /// Whether to reverse bars order for each channel
+    /// </summary>
+    [Option('e', "reverse-order", Required = false, HelpText = "Reverse order of bars (true or false)")]
+    public bool? ReverseOrder { get; set; }
+    /// <summary>
+    /// Drawing direction
+    /// </summary>
+    [Option('o', "drawing-direction", Required = false, HelpText = "Drawing direction, one of: topbottom, bottomtop, leftright, rightleft")]
+    public DrawingDirection? Direction { get; set; }
+    /// <summary>
+    /// The size of spaces between elements
+    /// </summary>
+    [Option('t', "items-offset", Required = false, HelpText = "Offset between items (0-20)")]
+    public uint? ItemsOffset { get; set; }
+    /// <summary>
+    /// Roundness of items (0 - square, 1 - round)
```

Comment:
```suggestion
    /// Roundness of items (0 - square, 100 - round)
```

Looks like this should be 100?

Reply:
right, here it's 100